frontend/src/metabase/dashboard/containers/DashboardHeader.styled.jsx

```diff
@@ -16,6 +16,7 @@ export const DashboardHeaderButton = styled(Button)`
   height: 2rem;
   width: 2rem;
   color: ${props => (props.isActive ? color("brand") : color("text-dark"))};
+  font-size: 1rem;
```

Comment:
Ensure this doesn't get overridden by
https://github.com/metabase/metabase/blob/353e7097540859dc027f7bc4c9738f387297e664/frontend/src/metabase/css/dashboard.css#L276-L280

which cause the line-height to be larger than the non-fullscreen version